src/framework/preferences/js/Enactors.js

```diff
@@ -427,4 +427,244 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+            }
+            else {
+                var backgroundColorToSet = "rgba(" + backgroundColor[0] + ", " + backgroundColor[1] + ", " + backgroundColor[2] + ", " + backgroundColor[3] + ")";
+            }
+            $(this).css("background-color", backgroundColorToSet);
+        });
+    };
+
+    fluid.prefs.enactor.blueColorFilter.addMutationObserver = function (applyFilter) {
+        // Uses document ready, because the script is added before the body
+        $(document).ready(function (){
+            var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+            if (isSafari) {
+                var observer = new WebKitMutationObserver(function(mutations) {
+                    mutations.forEach(function (mutation) {
+                        if (mutation.type === "childList") {
+                            // Changes in HTMLHtmlElement can be just hovering over elements, which should not trigger applyFilter
+                            if (changedDOMFlag === false && mutation.target != "[object HTMLHtmlElement]") {
+                                changedDOMFlag = true;
+                                applyFilter();
+                            }
+                        }
+                    });
+                    changedDOMFlag = false;
+                });
+            }
+            else {
+                var observer = new MutationObserver(function(mutations) {
+                    mutations.forEach(function (mutation) {
+                        if (mutation.type === "childList") {
+                            // Changes in HTMLHtmlElement can be just hovering over elements, which should not trigger applyFilter
+                            if (changedDOMFlag === false && mutation.target != "[object HTMLHtmlElement]") {
+                                changedDOMFlag = true;
+                                applyFilter();
+                            }
+                        }
+                    });
+                    changedDOMFlag = false;
+                });
+            }
+
+            var target = document.body;
+            var config = {
+                childList : true,
+                subtree : true
+            };
+
+            observer.observe(target, config);
+        });
+    };
+
+    /* Add a mutationObserver to track when the UI panel in its responsive version is opened.
+       It then adds and removes a class to the hide/show button to style it properly. */
+    $(document).ready(function (){
+        var isSafari = /^((?!chrome|android).)*safari/i.test(navigator.userAgent);
+        function observerFunction() {
+            var openedResponsivePanelCheck = $(".flc-slidingPanel-panel.flc-prefsEditor-iframe").attr("aria-expanded");
+            var panelButtons = $(".fl-prefsEditor-buttons");
+            if (openedResponsivePanelCheck == "true" && $(window).width() < 640) {
+                panelButtons.addClass("flc-prefsEditor-opened-panel-buttons");
```

Comment:
1. The class name should not be hardcoded here. It should be from a component option.
2. The infusion naming convention of when to use ```flc-``` and ```fl-``` prefixes:
- the class names starting with flc- are used to identify the elements to UI Options;
- the class names starting with fl- are used for visual styling.
The css class here is for visual styling so please use prefix ```fl-```. Also, the css styling classes are usually defined in components' options.styles block. An example: https://github.com/fluid-project/infusion/blob/master/src/components/overviewPanel/js/OverviewPanel.js#L138